multipatch_analysis/experiment_list.py

```diff
@@ -329,9 +329,10 @@ def distance_plot(self, pre_types=None, post_types=None, connection_types=None,
             name = ("%s->%s "%(','.join(pre_strs), ','.join(post_strs)))
         return distance_plot(connected, distance=probed, plots=plots, color=color, name=name, window=40e-6, spacing=40e-6)
 
-    def matrix(self, rows, cols, size=50, header_color='k', no_data_color=0.9, mode='connectivity'):
+    def matrix(self, rows, cols, size=50, header_color='k', no_data_color=0.9, mode='connectivity', title=None):
         w = pg.GraphicsLayoutWidget()
         w.setRenderHints(w.renderHints() | pg.QtGui.QPainter.Antialiasing)
+        w.setWindowTitle(title)
```

Comment:
`if title is not None`

Reply:
or what about just setting the default to Connectivity Matrix?

Reply:
+1